# translation of gl.po to galician
# translation of update-manager-gl.po to galician
# This file is distributed under the same license as the update-manager package.
#  Copyright (c) 2004 Canonical
#                2004 Michiel Sikkes
# Mar Castro <mariamarcp@gmail.com>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: gl\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-10-08 15:29-0700\n"
"PO-Revision-Date: 2019-06-18 17:53+0000\n"
"Last-Translator: Brian Murray <brian@ubuntu.com>\n"
"Language-Team: galician\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:47+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Servidor situado en %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Servidor principal"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Servidores personalizados"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "Non e posíbel calcular a entrada do sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Non é posíbel localizar ningún ficheiro de paquete; se cadra este non é un "
"disco de Ubuntu ou a arquitectura non é a correcta."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "Non foi posíbel engadir o CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Produciuse un erro ao engadir o CD e hai que cancelar a anovación. Informe "
"deste erro se se trata dun CD correcto de Ubuntu.\n"
"\n"
"A mensaxe de erro foi:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Retirar o paquete en mal estado"
msgstr[1] "Retirar os paquetes en mal estado"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"O paquete «%s» ten un estado inconsistente e precisa ser reinstalado pero "
"non se atopa ningún ficheiro para facelo. Quere eliminar este paquete e "
"continuar?"
msgstr[1] ""
"Os paquetes «%s» teñen un estado inconsistente e precisan ser reinstalados "
"pero non se atopa ningún ficheiro para facelo. Quere eliminar estes paquetes "
"e continuar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "O servidor quizais estea saturado"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Paquetes rotos"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"O seu sistema contén paquetes rotos que non poden ser arranxados con este "
"software. Por favor, arránxeos primeiro usando Synaptic ou apt-get antes de "
"continuar."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Paquetes rotos"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""
"Produciuse un erro irresolúbel ao calcular a anovación.\n"
"\n"
" "

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""
"Isto debeuse, o máis seguro, a:\n"
" * Paquetes de software non oficiais non fornecidos por Ubuntu\n"
"Emprgue a ferramenta «ppa-purge» do paquete ppa-purge\n"
"para retirar software das PPA de Launchpad e tente\n"
"a anovación de novo.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""
"Isto debeuse a:\n"
" * Anovar a unha versión previa de Ubuntu\n"
"Isto é, moi posibelmente, iun problema transitorio;\n"
"tente de novo máis tarde.\n"

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr "Trátase, o máis seguro, dun problema temporal. Ténteo máis tarde."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""
"Se non se aplica nada disto, informe deste erro empregando a orde «ubuntu-"
"bug ubuntu-release-upgrader-core» nun terminal. "

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""
"Se desexa investigar isto vostede mesmo, os ficheiros de rexistros en «/var/"
"log/dist-upgrade» conteñen detalles sobre a anovación. En concreto, mire en "
"«main.log» e «apt.log»."

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "Non foi posíbel calcular a anovación"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "Produciuse un erro ao autenticar algúns paquetes"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Non foi posíbel autenticar algúns paquetes. Isto pode ser debido a un "
"problema transitorio na rede. Probe de novo máis tarde. Vexa abaixo unha "
"lista dos paquetes non autenticados."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"O paquete «%s» está marcado para a súa eliminación pero está na lista negra "
"de paquetes eliminábeis."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "O paquete esencial «%s» está marcado para a súa eliminación"

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Tentando instalar a versión da lista negra «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "Non foi posíbel instalar «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Non foi posíbel instalar un paquete requirido. Informe deste erro usando "
"«ubuntu-bug ubuntu-release-upgrade-core» no seu terminal."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "Non foi posíbel determinar o meta-paquete"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"O sistema non contén os paquetes %s nin %s e non foi posíbel detectar a "
"versión de Ubuntu en execución.\n"
" Instale un dos paquetes anteriores usando synaptic ou apt-get antes de "
"continuar."

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "Lendo a caché"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "Foi imposíbel obter un bloqueo exclusivo"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Isto normalmente significa que outro xestor de paquetes (como apt-get ou "
"aptitude) xa se está executando. Peche este aplicativo antes de continuar."

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "A actualización mediante unha conexión remota non é posíbel"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Está executando a anovación a través dunha conexión SSH cunha interface que "
"non é compatíbel con esta. Por favor, tente unha anovación en modo de texto "
"con «do-release-upgrade».\n"
"\n"
"A anovación vai ser cancelada agora. Por favor, ténteo sen ssh."

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "Continuar a execución con SSH?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Esta sesión semella estar executandose baixo SSH. Na actualidade non se "
"recomenda facer unha anovación a través de ssh porque no caso de fracaso, é "
"máis difícil de recuperar.\n"
"\n"
"Se continúa, iniciarase un daemon ssh adicional no porto «%s».\n"
"Desexa continuar?"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "Iniciando un sshd adicional"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Para que resulte máis doada a recuperación no caso de fallo, iniciarase un "
"sshd adicional no porto «%s». Se algo sae mal co ssh en execución, aínda "
"poderá conectarse co adicional.\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Se executa unha devasa, pode necesitar abrir este porto temporalmente. Como "
"isto é potencialmente perigoso, non se fai automaticamente. Pode abrir o "
"porto con, p.ex.:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "Non é posíbel anovar"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Con esta ferramenta non se pode anovar de «%s» a «%s»"

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""
"A instalación de python3 está estragada. Arranxe a ligazón simbólica «/usr/"
"bin/python3»."

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "Non é posíbel escribir en «%s»"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Non é posíbel escribir no directorio do sistema «%s» deste sistema. A "
"actualización non pode continuar.\n"
"Comprobe que se pode escribir no directorio do sistema."

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "Incluír as actualizacións máis recentes desde a Internet?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"O sistema de anovación pode descargar automaticamente as actualizacións máis "
"recentes da Internet e instalalas durante a anovación. Se ten unha conexión "
"de banda larga, isto é moi recomendábel.\n"
"\n"
"A anovación levará máis tempo, mais unha vez rematada, o seu sistema estará "
"completamente actualizado. Pode escoller non facelo, mais debería instalar "
"as actualizacións máis recentes antes de actualizar o sistema.\n"
"Se dis que «non» aquí, non se usará a conexión para nada."

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""
"Non indicado para usuarios normais durante o estadio de desenvolvemento da "
"versión %s"

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "desactivado na anovación a %s"

#: ../DistUpgrade/DistUpgradeController.py:806
msgid "No valid sources.list entry found"
msgstr "Non foi posíbel atopar unha sources.list válida"

#: ../DistUpgrade/DistUpgradeController.py:807
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Non foi posíbel atopar unha entrada de %s durante a análise da información "
"do repositorio.\n"
"\n"
"Quizais unha anovación non dea resultados.\n"
"\n"
"Desexa continuar?"

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "Non se atopou ningunha réplica correcta"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Mentres se analizaba a información dos seus repositorios non se atopou "
"ningunha entrada dun servidor espello para a anovación. Isto pode suceder se "
"executa un servidor espello interno ou se a información do servidor espello "
"non está actualizada.\n"
"\n"
"Desexa reescribir o seu ficheiro «sources.list» igualmente? Se escolle «Si» "
"actualizaranse todas as entradas «%s» a «%s».\n"
"Se escolle «Non» cancelarase a anovación."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "Xerar as fontes predeterminadas?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Unha vez examinado «sources.list», non se atopou ningunha entrada correcta "
"para «%s».\n"
"\n"
"Deberíanse engadir as entradas por omisión de «%s»? Se escolle «Non», "
"anularase a anovación."

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "A información do repositorio non é correcta"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"A anovación da información do respositoriu produciu un ficheiro erróneo, "
"polo que se está a iniciar un proceso de informe de erros."

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"A anovación da información do repositorio obtivo un ficheiro incorrecto. "
"Para informar deste fallo instale apport e execute «apport-bug ubuntu-"
"release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "Fontes de terceiros desactivadas"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Desactiváronse algunhas entradas de terceiras partes no seu sources.list. "
"Pódeas volver a activar coa ferramenta «software-properties» ou co xestor de "
"paquetes."

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paquete en estado inconsistente"
msgstr[1] "Paquetes en estado inconsistente"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"O paquete «%s» ten un estado inconsistente e precisa ser reinstalado pero "
"non se atopa ningún ficheiro para facelo. Reinstale o paquete manualmente ou "
"elimíneo do sistema."
msgstr[1] ""
"Os paquetes «%s» teñen un estado inconsistente e precisan ser reinstalados "
"pero non se atopa ningún ficheiro para facelo. Reinstale os paquete "
"manualmente ou elimíneos do sistema."

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "Produciuse un erro durante a actualización"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Ocorreu un problema durante a actualización. Normalmente é debido a algún "
"tipo de problema na rede. Por favor, comprobe a súa conexión de rede e "
"vólvao a intentar."

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr "Partición do sistema EFI (ESP) non usábel"

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"A súa Partición do Sistema EFI (ESP) non está montada en /boot/efi. "
"Asegúrese que está correctamente configurada e ténteo de novo."

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"A Partición do Sistema EFI (ESP) montada en /boot/efi non é escribíbel. "
"Monte esta partición en modo lectura-escritura e ténteo de novo."

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "Non hai espazo abondo no disco"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"A anovación abortou. A anovación precisa un total de {str_total} espazo "
"libre no disco «{str_dir}». Libere polo menos {str_needed} adicionais de "
"espazo no disco en «{str_dir}». {str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Elimine paquetes temporais de instalacións anteriores usando «sudo apt "
"clean»."

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"Pode eliminar vellos kernels usando «sudo apt autoremove» e tamén pode "
"estabelecer COMPRESS=xz in /etc/initramfs-tools/initramfs.conf para reducir "
"o tamaño da seus initramfs."

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Baleire o lixo e elimine os paquetes temporais de instalacións antigas "
"usando «sudo apt-get clean»."

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr "Reiniciar para limpar os ficheiros de /temp."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2047
msgid "Calculating the changes"
msgstr "Calculando os cambios"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "Desexa comezar a anovación?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""
"As actualizacións de seguranza de Livepatch non están dispoñíbeis para "
"Ubuntu %s. Se anova, Livepatch será desactivado."

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "Anulouse a anovación"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Agora vaise cancelar a anovación e o sistema volverá ao seu estado orixinal. "
"Pode retomar a actualización máis adiante."

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "Non se puideron descargar as anovacións"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Abortouse a actualización. Comprobe a súa conexión a Internet ou medio de "
"instalación e ténteo de novo. Tódolos ficheiros descargados serán mantidos."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "Produciuse un erro durante a remisión"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1551
msgid "Restoring original system state"
msgstr "Restaurando o estado orixinal do sistema"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "Non se puideron instalar as anovacións"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Abortouse a anovación. O seu sistema pode estar nun estado non utilizábel. "
"Vaise executar agora a recuperación (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Informe de este erro usando o seu navegador http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug e anexe os ficheiros existentes en /"
"var/log/dist-upgrade/ ao informe do erro.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Cancelouse a actualización. Comprobe a súa conexión a Internet ou o "
"dispositivo de instalación e volva a tentalo. "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "Buscando paquetes obsoletos"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "Retirar os paquetes obsoletos?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Conservar"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "_Retirar"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Ocorreu algún problema durante a limpeza. Por favor, vexa a mensaxe inferior "
"para obter máis información. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Required depends is not installed"
msgstr "Non está instalado depends, que é necesario"

#: ../DistUpgrade/DistUpgradeController.py:1590
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "A dependencia requirida «%s» non está instalada. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1857
#: ../DistUpgrade/DistUpgradeController.py:1947
msgid "Checking package manager"
msgstr "Comprobando o xestor de paquetes"

#: ../DistUpgrade/DistUpgradeController.py:1863
#: ../DistUpgrade/DistUpgradeController.py:1869
msgid "Preparing the upgrade failed"
msgstr "Fallou a preparación da anovación"

#: ../DistUpgrade/DistUpgradeController.py:1864
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Produciuse un fallo ao preparar o sistema para a anovación polo que estase "
"iniciando un proceso de informe de erro."

#: ../DistUpgrade/DistUpgradeController.py:1870
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Preparando o sistema para a anovación fallida. Para informar dun erro "
"instale apport e execute «apport-bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1887
#: ../DistUpgrade/DistUpgradeController.py:1897
msgid "Getting upgrade prerequisites failed"
msgstr "Fallou a preparación da anovación"

#: ../DistUpgrade/DistUpgradeController.py:1888
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"O sistema non cumpre os prerequisitos para a súa anovación. A anovación "
"cancelarase agora e restaurarase o estado orixinal do sistema.\n"
"\n"
"Ademais, estase iniciando un proceso de informe de erro."

#: ../DistUpgrade/DistUpgradeController.py:1898
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"O sistema non foi quen de obter os requisitos previos para a anovación. A "
"anovación cancelarase agora e restabelecerase o estado orixinal do sistema.\n"
"\n"
"Para informar dun fallo instale apport e execute «apport-bug ubuntu-release-"
"upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Updating repository information"
msgstr "Actualizando a información do repositorio"

#: ../DistUpgrade/DistUpgradeController.py:1938
msgid "Failed to add the cdrom"
msgstr "Produciuse un fallo ao engadir o CDROM"

#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Sorry, adding the cdrom was not successful."
msgstr "Sentímolo, non foi posíbel engadir o CDROM"

#: ../DistUpgrade/DistUpgradeController.py:1974
msgid "Invalid package information"
msgstr "Información de paquete incorrecta"

#: ../DistUpgrade/DistUpgradeController.py:1975
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Despois de anovar a súa información de paquetes, non foi posíbel atopar o "
"paquete esencial «%s». Isto pode deberse a que non ten as réplicas oficiais "
"na lista de fontes de software ou porque hai unha carga excesiva na réplica "
"que está empregando. Bótelle un ollo á lista de fontes de software "
"configurada actualmente en /etc/apt/sources.list.\n"
"No caso de que sexa por mor dunha réplica saturada, debería anovar un pouco "
"máis tarde."

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2103
msgid "Fetching"
msgstr "Obtendo"

#: ../DistUpgrade/DistUpgradeController.py:2011
#: ../DistUpgrade/DistUpgradeController.py:2032
#: ../DistUpgrade/DistUpgradeController.py:2053
#: ../DistUpgrade/DistUpgradeController.py:2107
msgid "Upgrading"
msgstr "Anovando"

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "Upgrade infeasible"
msgstr "Anovación non realizábel"

#: ../DistUpgrade/DistUpgradeController.py:2014
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""
"Non foi posíbel completar a anovación. Producíronse erros durante o proceso."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2035
msgid "Upgrade incomplete"
msgstr "Anovación incompleta"

#: ../DistUpgrade/DistUpgradeController.py:2036
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""
"A anovación completouse parcialmente pero producíronse erros durante o "
"proceso."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2058
#: ../DistUpgrade/DistUpgradeController.py:2109
#: ../DistUpgrade/DistUpgradeController.py:2115
#: ../DistUpgrade/DistUpgradeController.py:2126
msgid "Upgrade complete"
msgstr "Completouse a anovación"

#: ../DistUpgrade/DistUpgradeController.py:2059
#: ../DistUpgrade/DistUpgradeController.py:2110
#: ../DistUpgrade/DistUpgradeController.py:2116
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "A actualización completouse mais producíronse erros durante o proceso."

#: ../DistUpgrade/DistUpgradeController.py:2078
msgid "System upgrade is complete."
msgstr "Completouse a anovación do sistema."

#: ../DistUpgrade/DistUpgradeController.py:2127
msgid "The partial upgrade was completed."
msgstr "Completouse a anovación parcial do sistema."

#: ../DistUpgrade/DistUpgradeController.py:2155
msgid "Lock screen disabled"
msgstr "Bloqueo da pantalla desactivado"

#: ../DistUpgrade/DistUpgradeController.py:2156
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""
"O bloqueo da pantalla foi desactivado e permanecerá así até que reinicie."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Non foi posíbel atopar as notas da versión"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Poida que o servidor estea sobrecargado. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Non foi posíbel descargar as notas da versión"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Por favor, comprobe a conexión á Internet."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "autenticar «%(file)s» contra «%(signature)s» "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "a extraer «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "Non foi posíbel executar a ferramenta de anovación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Isto semella ser un erro na ferramenta de anovación. Informe deste erro "
"usando a orde «ubuntu-bug ubuntu-release-upgrader-core»."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Sinatura da ferramenta de anovación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Ferramenta de anovación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Erro ao obter"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Fallou a obtención da anovación. Pode haber un problema coa rede. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Fallou a autenticación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Fallou a autenticación da anovación. Pode haber un problema coa rede ou co "
"servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Erro ao extraer"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Fallou a extracción da anovación. Pode haber un problema coa rede ou co "
"servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Fallou a verificación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Fallou a verificación da anovación. Pode haber un problema coa rede ou co "
"servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "Non foi posíbel executar a anovación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Isto a miúdo é causado por un sistema onde /tmp está montado como parámetro "
"«noexec». Remónte a partición sen o parámetro noexec antes de realizar a "
"anovación."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "A mensaxe do erro é «%s»."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Anovar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Cancelar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Notas da versión"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Anovar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Descargando os ficheiros dos paquetes adicionais..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Ficheiro %s de %s a %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Ficheiro %s de %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Por favor, introduza «%s» na unidade «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Cambio de medio"

#: ../DistUpgrade/DistUpgradeQuirks.py:230
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"É posíbel que o hardware gráfico non sexa totalmente compatíbel co Ubuntu "
"14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:232
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"O hardware de gráficos non admite de todo a execución do ambiente de "
"escritorio «unity». Pode que, unha vez rematada a actualización, o ambiente "
"sexa moi lento. Recomendamos que, de momento, fique coa versión TLS. Para "
"máis información, consulte https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D. Desexa proseguir coa actualización?"

#: ../DistUpgrade/DistUpgradeQuirks.py:256
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"O seu hardware de gráficos non é completamente compatíbel con Ubuntu 12.04 "
"LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:258
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"A compatibilidade en Ubuntu 12.04 LTS para o seu hardware de gráficos Intel "
"está limitada e pode atopar problemas trala actualización. Para ter máis "
"información vexa https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx "
"¿Desexa continuar coa anovación?"

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"A anovación pode reducir os efectos do escritorio, o rendemento nos xogos e "
"noutros programas que fagan un emprego intensivo dos gráficos."

#: ../DistUpgrade/DistUpgradeQuirks.py:282
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Este computador emprega o controlador gráfico «fglrx» de AMD. Non se conta "
"cun controlador que funcione co seu hardware en Ubuntu 10.04 LTS.\n"
"\n"
"Está seguro de que desexa continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:321
msgid "Sorry, no more upgrades for this system"
msgstr "Desculpe, mais non hai máis anovacións para este sistema"

#: ../DistUpgrade/DistUpgradeQuirks.py:322
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""
"Non haberá mais edicións de Ubuntu para a arquitectura «i386» deste "
"sistema.\n"
"\n"
"As actuacións para Ubuntu %s continuarán até %s."

#: ../DistUpgrade/DistUpgradeQuirks.py:337
msgid "No ARMv6 CPU"
msgstr "Non é unha CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:338
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"O seu sistema emprega unha CPU ARM que é máis antiga que a arquitectura "
"ARMv6. Todos os paquetes en karmic foron construídos coas optimizacións que "
"require ARMv6 como arquitectura mínima. Non é posíbel anovar o seu sistema a "
"unha versión de Ubuntu con esta arquitectura."

#: ../DistUpgrade/DistUpgradeQuirks.py:358
msgid "No init available"
msgstr "O «daemon» init non está dispoñíbel"

#: ../DistUpgrade/DistUpgradeQuirks.py:359
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Semella que o seu sistema é un contorno virtualizado sen un daemon init (un "
"Linux-VServer, p.ex.). Ubuntu 10.04 LTS non pode funcionar neste tipo de "
"contorno, polo que require unha actualización da configuración da súa "
"máquina virtual.\n"
"\n"
"Está seguro de que desexa continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:400
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:401
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:519
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "Produciuse un fallo conectando coa tenda de snaps"

#: ../DistUpgrade/DistUpgradeQuirks.py:520
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:535
msgid "Connection to Snap Store failed"
msgstr "Produciuse un fallo conectando coa tenda de snaps"

#: ../DistUpgrade/DistUpgradeQuirks.py:536
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"O sistema non ten conexión coa tenda de snaps. Para ter a mellor experiencia "
"nas anovacións asegúrese de que o sistema pode conectar con api.snapcraft."
"io.\n"
"Desexa continuar coa anovación?"

#: ../DistUpgrade/DistUpgradeQuirks.py:545
msgid "Outdated snapd package"
msgstr "O paquete snapd está obsoleto"

#: ../DistUpgrade/DistUpgradeQuirks.py:546
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"O sistema non dispón da última versión do snapd. Actualice a versión deste "
"paquete para mellorar a experiencia de anovación.\n"
"Desexa continuar coa anovación?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:569
msgid "Calculating snap size requirements"
msgstr "A calcular os requisitos de tamaño dos snaps"

#: ../DistUpgrade/DistUpgradeQuirks.py:600
msgid "Processing snap replacements"
msgstr "A procesar as substitucións de snaps"

#: ../DistUpgrade/DistUpgradeQuirks.py:606
#, python-format
msgid "refreshing snap %s"
msgstr "a refrescar o snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:610
#, fuzzy, python-format
msgid "removing snap %s"
msgstr "a refrescar o snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:613
#, python-format
msgid "installing snap %s"
msgstr "a instalar o snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:642
msgid "PAE not enabled"
msgstr "A PAE non está activada"

#: ../DistUpgrade/DistUpgradeQuirks.py:643
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"O seu sistema utiliza unha CPU que non ten activada a PAE. Ubuntu só é "
"compatíbel con sistemas non-PAE até Ubuntu 12.04. Para anovar a unha versión "
"máis recente de Ubuntu, debe activar a PAE (se é posíbel) vexa:\n"
" http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:976
msgid "Checking for installed snaps"
msgstr "Comprobando os snaps instalados"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Usar a ruta dada para buscar un CD-ROM con paquetes de actualización"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Usar a interface gráfica. Dispoñíbeis neste momento: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*OBSOLETO* esta opción será ignorada"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Realizar só unha anovación parcial (sen reescribir o sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Desactivar a compatibilidade de pantalla GNU"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Estabelecer datadir"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Anovar á versión de desenvolvemento"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Rematou a obtención"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Obtendo o ficheiro %li de %li a %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Restan aproximadamente %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Obtendo o ficheiro %li de %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Aplicando os cambios"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "problemas de dependencias - déixase sen configurar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Non foi posíbel instalar «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"A anovación continuará mais o paquete «%s» podería estar instalado "
"incorrectamente. Considere o envío dun informe de erro sobre iso."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Substituír o ficheiro de configuración personalizado\n"
"«%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Perderá as modificacións feitas neste ficheiro de configuración se escolle "
"substituílo por unha versión máis nova."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "Non se atopou a orde «diff»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Produciuse un erro fatal"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informe deste fallo (se aínda non o fixo) e inclúa os ficheiros /var/log/"
"dist-upgrade/main.log e /var/log/dist-upgrade/apt.log no informe. A "
"actualización foi cancelada.\n"
"O seu ficheiro sources.list orixinal foi gardado en /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl+C premido"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Isto interromperá a operación e podería deixar o sistema estragado. Seguro "
"que quere facelo?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr "Para previr a perda de datos peche todos os aplicativos e documentos."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Sen máis asistencia por parte de Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Desactualizar (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Retirar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Xa non son necesarios (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Instalar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Anovar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr "Cancelar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr "Continuar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Mostrar a diferenza >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Agochar a diferenza"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Erro"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Pechar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Mostrar o terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Agochar o terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Información"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Iniciar a anovación"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detalles"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Sen máis asistencia (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Retirar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Retirar (instalado automaticamente) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Instalar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Anovar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Cómpre reiniciar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Reinicie o sistema para completar a anovación</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Reiniciar agora"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Anular a anovación en curso?</big></b>\n"
"\n"
"O sistema podería quedar inutilizado se anula a anovación. Recomendámoslle "
"encarecidamente que continúe a anovación."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Anular a anovación?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li día"
msgstr[1] "%li días"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hora"
msgstr[1] "%li horas"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuto"
msgstr[1] "%li minutos"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li segundo"
msgstr[1] "%li segundos"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Esta descarga durará arredor de  %s nunha conexión DSL a 1Mbit e sobre  %s "
"nunha conexión de módem a  56k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Esta descarga levará aproximadamente %s coa súa conexión. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Preparando a anovación"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Obtendo as canles de software novas"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Obtendo os paquetes novos"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Instalando as anovacións"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Limpando"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paquete instalado non seguirá tendo asistencia de Canonical. "
"Aínda así, pode obter asistencia desde a comunidade."
msgstr[1] ""
"%(amount)d paquetes instalados non seguirán tendo asistencia de Canonical. "
"Aínda así, pode obter asistencia desde a comunidade."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Hase retirar %d paquete."
msgstr[1] "Hanse retirar %d paquetes."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Hase instalar %d paquete novo."
msgstr[1] "Hanse instalar %d paquetes novos."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Hase anovar %d paquete."
msgstr[1] "Hanse anovar %d paquetes."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Hai que descargar un total de %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"A instalación da anovación pode levar varias horas. En canto remate a "
"descarga non será posíbel cancelar o proceso."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Obter e instalar a anovación pode levar varias horas. En canto remate a "
"descarga non será posíbel cancelar o proceso."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "A eliminación dos paquetes pode levar varias horas. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "O software deste computador está actualizado."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Non hai anovacións dispoñíbeis para o seu sistema. Anularase a anovación."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Cómpre reiniciar"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "A anovación finalizou e cómpre reiniciar. Quere facelo agora?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progreso: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informe deste fallo e inclúa os ficheiros /var/log/dist-upgrade/main.log e /"
"var/log/dist-upgrade/apt.log no informe. A actualización foi cancelada.\n"
"O seu ficheiro sources.list orixinal gardouse en /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Interrompendo"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Degradado:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Para continuar prema [INTRO]"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Continuar [sN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detalles [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "s"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Sen máis asistencia: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Retirar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Instalar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Anovar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Continuar [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Para concluír a anovación compre reiniciar.\n"
"Se selecciona «s» o sistema reiniciarase."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Descargando o ficheiro %(current)li de %(total)li a %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Descargando o ficheiro %(current)li de %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Mostrar o progreso de cada ficheiro individual"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Anular a anovación"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Continuar a anovación"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Cancelar a anovación en curso?</big></b>\n"
"\n"
"O sistema podería quedar nun estado non utilizábel se anula a anovación. "
"Recomendámoslle encarecidamente que continúe a anovación."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Iniciar a anovación"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Substituír"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Diferenza entre os ficheiros"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Informar dun fallo"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Continuar"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Comezar a anovación?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reinicie o sistema para completar a anovación</big></b>\n"
"\n"
"Garde o seu traballo antes de continuar."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Anovación da distribución"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>A anovar Ubuntu á versión %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Estabelecendo novas canles de software"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Reiniciando o computador"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "A_novar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Existe unha nova versión de Ubuntu dispoñíbel. Desexa anovar?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Non anovar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Preguntarme máis tarde"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Si, anovar agora"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Vostede rexeitou a anovación ao novo Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Pode facelo no futuro abrindo o Actualizador de software e premendo en "
"«Anovar»."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Realizar unha anovación de versión"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Para anovar Ubuntu, é preciso que se autentifique."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Realizar unha anovación parcial"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Para realizar unha anovación parcial, é preciso que se autentifique."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Mostrar a versión e saír"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Directorio que contén os ficheiros de datos"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Execute a interface gráfica indicada"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Executando unha anovación parcial"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Descargando a ferramenta de anovación"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Se usa a última versión con asistencia, anove á versión de desenvolvemento"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Tente anovarse á última versión usando o anovador desde $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Executar nun modo de anovación especial.\n"
"Actualmente existen «desktop» para actualizacións normais dun sistema de "
"escritorio e «server» para sistemas servidores."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Comprobar só se está dispoñíbel unha nova versión da distribución e informar "
"do resultado mediante o código de saída."

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""
"Proba a anovar coas réplicas e repositorios de terceiros activados en "
"troques de anulalos como comentarios."

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "As opcións  --devel-release e --proposed"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "exclúense mutuamente. Use só unha delas."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "Buscando novas publicacións de Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "A súa versión de Ubuntu xa non está asistida."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Para obter máis información sobre esta anovación, visite:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr "está configurado para nuncia, polo que non é posíbel anovar."

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr "Non hai dispoñíbel ningunha versión en desenvolvemento dunha LTS"

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr "Para anovar á última edición en desenvolvemento non LTS "

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr "configure Prompt=normal en /etc/update-manager/release-upgrades."

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr "As anovacións á versión de desenvolvemento están unicamente "

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr "dispoñíbeis desde a última versión con asistencia."

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr "Non se atopou ningunha versión nova."

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "Non é posíbel actualizar a versión neste momento"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Non é posíbel realizar a actualización de versión neste momento, probe de "
"novo máis tarde. O servidor informou: «%s»"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "Unha versión nova «%s» dispoñíbel"

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Execute «do-release-upgrade» para anovarse."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""
"Instale todas as actualizacións dispoñíbeis para esta versión antes de "
"anovar."

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""
"Non reiniciou logo de actualizar un paquete que require un reinicio. "
"Reinicie antes de anovar."

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "Actualizador de software"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Está dispoñíbel unha anovación á %(version)s de Ubuntu"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Vostede rexeitou a anovación a Ubuntu %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Comprobar se resulta posíbel anovar á versión en desenvolvemento máis recente"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "Engadir a saída de depuración"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Un problema irresolúbel ocorreu mentres se calculaba a anovación:\n"
#~ "%s\n"
#~ " Isto pode ser causado por:\n"
#~ " * Anovar desde unha versión de prelanzamento de Ubuntu\n"
#~ " * Executar a versión actual de prelanzamento de Ubuntu\n"
#~ " * Paquetes de software non oficiais non fornecidos por Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "O seu sistema non contén os paquetes ubuntu-desktop, kubuntu-desktop ou "
#~ "edubuntu-desktop e non foi posíbel detectar cal é a versión de ubuntu que "
#~ "se está a executar.\n"
#~ " Instale un dos paquetes mencionados usando synaptic ou apt-get antes de "
#~ "continuar."

#~ msgid "Sandbox setup failed"
#~ msgstr "A configuración da zona de probas fallou"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Non foi posíbel crear o contorno da zona de probas."

#~ msgid "Sandbox mode"
#~ msgstr "Modo zona de probas"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Esta anovación está executándose en modo «caixa de area» (proba). Todos "
#~ "os cambios escribiránse en «%s» e perderanse no seguinte \n"
#~ "\n"
#~ "Os cambios no sistema de ficheiros serán permanentes ate que reinicie."

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "A súa instalación de Python é defectuosa. Arranxe a ligazón simbólica «/"
#~ "usr/bin/python»."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "O paquete «debsig-verify» está instalado"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "A anovación non pode continuar con ese paquete instalado.\n"
#~ "Retíreo  con synaptic ou «apt-get remove debsig-verify» primeiro e faga a "
#~ "anovación de novo."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Non se atopou unha entrada na réplica para a anovación ao examinar a "
#~ "información do repositorio. Isto pode acontecer se se executa unha "
#~ "réplica interna ou se a información da réplica estivera anticuada.\n"
#~ "\n"
#~ "Quere substituír «sources.list» igual? Se escoller «Si» aquí, "
#~ "actualizaranse todas as entradas «%s» a «%s».\n"
#~ "Se seleccionar «Non», cancelarase a actualización."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Cancelouse a anovación. A anovación precisa un total de %s de espazo "
#~ "libre no disco «%s». Libere cando menos %s de espazo on disco «%s». "
#~ "Libere o lixo e elimine os paquetes temporais das instalacións anterioers "
#~ "usando «sudo apt-get clean»."

#~ msgid "evms in use"
#~ msgstr "evms en uso"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "O sistema emprega o xestor de volumes «evms» en /proc/mounts. O software "
#~ "«evms» non ten asistencia; apágueo e execute de novo a anovación ao "
#~ "rematar."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Este computador emprega o controlador gráfico «nvidia» de NVIDIA. Non se "
#~ "conta cun controlador que funcione co seu hardware en Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Está seguro de que desexa continuar?"

#~ msgid "No i686 CPU"
#~ msgstr "Non é unha CPU i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "O seu sistema usa unha CPU i586 ou unha CPU que non ten a extensión "
#~ "«cmov». Todos os paquetes foron construídos con optimizacións que "
#~ "requiren i686 como arquitectura mínima. Non é posíbel actualizar o seu "
#~ "sistema á nova versión de Ubuntu con este hardware."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Anovación da área de probas usando aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "O test de anovación con área de probas aufs devolve superposición"

#~ msgid "No new release found"
#~ msgstr "Non se atopou unha versión nova"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Requírese autenticación para levar a cabo unha anovación de versión"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Requírese autenticación para levar a cabo unha anovación parcial"

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 13.04."
#~ msgstr "O seu hardware gráfico podería non ser compatíbel con Ubuntu 13.04."

#~ msgid "<b><big>Upgrading Ubuntu to version 13.04</big></b>"
#~ msgstr "<b><big>Anovando Ubuntu á versión 13.04</big></b>"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Produciuse un erro sen solución ao calcular a anovación.\n"
#~ "\n"
#~ " Isto pódese deber a:\n"
#~ " * Anovación a unha versión en desenvolvemento de Ubuntu\n"
#~ " * Execución da versión en desenvolvemento actual de Ubuntu\n"
#~ " * Paquetes de software non oficiais e non fornecidos por Ubuntu\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Estase anovando Ubuntu á versión 15.04</big></b>"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
#~ msgstr "<b><big>A anovar Ubuntu á versión 15.10</big></b>"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "Se ningún destes se pode aplicar, informe de este erro usando a orde "
#~ "«ubuntu-bug ubuntu-release-upgrade-core» no seu terminal."

#~ msgid "apt architecture is i386"
#~ msgstr "A arquitectura de apt é i386"

#~ msgid ""
#~ "Upgrades from 18.04 on the i386 architecture are not supported at this "
#~ "time."
#~ msgstr ""
#~ "Nesta altura non se admiten as actualizacións desde 18.04 na arquitectura "
#~ "i386."
